lib/wallet/rpc.js

```diff
@@ -449,6 +450,31 @@ class RPC extends RPCBase {
     return addrs;
   }
 
+  async getAddressInfo(args, help) {
+    if (help || args.length !== 1)
+      throw new RPCError(errs.MISC_ERROR, 'getaddressinfo "address"');
+
+    const valid = new Validator(args);
+    const addr = valid.str(0, '');
+    const address = parseAddress(addr, this.network);
+
+    const script = Script.decode(address.hash);
```

Comment:
I'm not sure how this function call is working correctly. We are passing a hash to`Script.decode`, but it expects to parse a script.

Reply:
You are right, good call. Fixed in d933131eb0e9a8f8c85a7aaba2919b5ad5f6554e